chapters/fa/chapter2/2.mdx

```diff
@@ -34,7 +34,7 @@
 <Youtube id="wVN12smEvqg"/>
 {/if}
 
-بگذارید با یک مثال کامل شروع کنیم که در آن نگاهی می‌اندازیم به آنچه که در پشت صحنه اتفاق می افتد هنگامی که این کد را در [Chapter 1](/course/chapter1) اجرا کردیم:
+بگذارید با یک مثال کامل شروع کنیم که در آن نگاهی می‌اندازیم به آنچه در پشت صحنه اتفاق افتاد وقتی این کد را در [فصل اول](/course/chapter1) اجرا کردیم:
```

Comment:
suggestion:
"بیایید با یک مثال کامل شروع کنیم و نگاهی به آنچه که در پشت صحنه به هنگام اجرای کد زیر در فصل ۱ رخ داد، بیاندازیم:"
instead of:
"بگذارید با یک مثال کامل شروع کنیم که در آن نگاهی می‌اندازیم به آنچه در پشت صحنه اتفاق افتاد وقتی این کد را در فصل اول اجرا کردیم:"

Reply:
I like the suggestion but not the comma placement. Went with this:

بگذارید با یک مثال کامل شروع کنیم. نگاهی می‌اندازیم به آنچه در پشت صحنه در اثر اجرای این قطعه کد در [فصل اول](/course/chapter1) رخ داد: